cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/transform/ValidatorTransform.java

```diff
@@ -127,6 +121,8 @@ void setUpInitialScript(TransformContext context, List<Validator> validators) th
     for (Validator validator : validators) {
       engine.put(validator.getValidatorName(), validator.getValidator());
     }
+    String jsonParseScript = "function parseJSON(input) { return JSON.parse(input); }";
```

Comment:
should do this the same way ScriptTransform does it.  I don't remember the exact reason, but there was some issue with JSON.parse().  I think it's not always available depending on the environment or something like that.